packages/esm-appointments-app/src/helpers/helper.ts

```diff
@@ -23,3 +24,47 @@ export const getServiceCountByAppointmentType = (
     .flat(1)
     .reduce((count, val) => count + val, 0);
 };
+
+function getAppointmentDuration(startTime = 0, endTime = 0) {
+  const diff = endTime - startTime;
+  var minutes = Math.floor(diff / 60000);
+  return minutes + 'Mins';
+}
+
+function formatAMPM(date) {
+  var hours = date.getHours();
+  var minutes = date.getMinutes();
+  var ampm = hours >= 12 ? 'PM' : 'AM';
```

Comment:
Should we translate AM and PM ?

Reply:
I can not call `useTranslations` inside these functions but maybe we can get a way to translate them.